testsuite/performance/pom.xml

```diff
@@ -35,13 +35,31 @@
         <server.version>${product.version}</server.version>
         <management.user/>
         <management.user.password/>
+        <jstat>false</jstat>
+        <keycloak.jstat>${jstat}</keycloak.jstat>
+        <infinispan.jstat>${jstat}</infinispan.jstat>
+        <lb.jstat>${jstat}</lb.jstat>
     </properties>
             
     <modules>
         <module>keycloak</module>
-        <module>load-balancer/wildfly-modcluster</module>
-        <module>infinispan</module>
         <module>tests</module>
     </modules>
     
+    <profiles>
+        <profile>
+            <id>cluster</id>
+            <modules>
+                <module>load-balancer/wildfly-modcluster</module>
```

Comment:
This causes a problem with module build order. LB module needs to be built before the tests.